bip-0173.mediawiki

```diff
@@ -273,11 +279,14 @@ The following strings have an invalid Bech32 checksum (with reason for invalidit
 * 0x20 + <tt>1nwldj5</tt>: HRP character out of range
 * 0x7F + <tt>1axkwrx</tt>: HRP character out of range
 * <tt>an84characterslonghumanreadablepartthatcontainsthenumber1andtheexcludedcharactersbio1569pvx</tt>: overall max length exceeded
-* <tt>pzry9x0s0muk</tt>: No separator character
-* <tt>1pzry9x0s0muk</tt>: Empty HRP
 * <tt>x1b4n0q5v</tt>: Invalid data character
 * <tt>li1dgmt3</tt>: Too short checksum
 * <tt>de1lg7wt</tt> + 0xFF: Invalid character in checksum
+* <tt>A1g7sgd8</tt>: The lowercase form is used when determining a character's value for checksum purposes.
+* <tt>A1G7SGD8</tt>: The lowercase form is used when determining a character's value for checksum purposes.
+
+The following are invalid Bech32 strings (with reason for invalidity):
+* <tt>pzry9x0s0muk</tt>: No separator character
```

Comment:
That's how it was to start. I moved it to a new section because I thought it was misleading in the current section because "No separator character" can not affect checksum validity because the separator char is not used to compute the checksum.

Reply:
Right, but that's to for several of the existing examples in that section (like HRP character out of range, or length exceeded.

I think they can be combined into a single section with examples of invalid Bech32 strings.